src/scripts.js

```diff
@@ -1,40 +1,106 @@
+const userRepo = new UserRepository(userData);
+// const currentUser = new User(userRepo.userList[0]);
+let currentUser;
```

Comment:
Do we need this to be a global variable? Can we put this within the function(s)?

Reply:
as of right now yes because it's being accessed by two separate functions